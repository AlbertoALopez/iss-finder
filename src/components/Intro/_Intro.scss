.iss-img {
  width: 100%;
}