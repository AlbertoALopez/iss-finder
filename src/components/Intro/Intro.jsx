import React from 'react';
import { Container, Header } from 'semantic-ui-react';
import './_Intro.scss';

const Intro = () => (
  <Container text>
    <img
      className="iss-img"
      src={require('./iss-large.png')}
    />
    <Header as="h2">Where in the world is the Internation Space Station?</Header>
    <p>Find out where over earth is the ISS</p>
  </Container>
);

export default Intro;
