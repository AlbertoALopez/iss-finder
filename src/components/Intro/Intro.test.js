import React from 'react';
import renderer from 'react-test-renderer';
import Intro from './Intro.jsx';

test('Intro renders correctly', () => {
  const component = renderer.create(
    <Intro />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
