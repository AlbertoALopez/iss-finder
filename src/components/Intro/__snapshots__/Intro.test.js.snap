exports[`test Intro renders correctly 1`] = `
<div
  className="ui text container">
  <img
    src="test-file-stub" />
  <h2
    className="ui header">
    Where in the world is the ISS?
  </h2>
  <p>
    Find out where the ISS is below.
  </p>
</div>
`;
